components/sup/src/util/users.rs

```diff
@@ -64,10 +64,10 @@ fn check_pkg_user_and_group(pkg_install: &PackageInstall) -> Result<Option<(Stri
             let current_user = current_user.unwrap();
             let current_group = current_group.unwrap();
 
-            if current_user == "root" {
+            if current_user == users::root_level_account() {
```

Comment:
If you were to convert `users::root_level_account()` to a `&'static str` then you'd call `current_user.as_slice()` here to do the comparison